ensenso_camera/package.xml

```diff
@@ -17,6 +17,7 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <build_depend>roslint</build_depend>
+  <depend>ensenso_sdk</depend>
```

Comment:
Right now there is no such ROS package called `ensenso_sdk`. You have to manually install the [Ensenso SDK](https://www.optonic.com/en/support/download/ensenso-sdk/) as described in the readme file.